dash/development/_py_components_generation.py

```diff
@@ -181,8 +215,12 @@ def generate_class_file(
     """
     import_string = (
         "# AUTO GENERATED FILE - DO NOT EDIT\n\n"
-        + "from dash.development.base_component import "
-        + "Component, _explicitize_args\n\n\n"
+        "import typing  # noqa: F401\n"
+        "import numbers # noqa: F401\n"
+        "import enum # noqa: F401\n"
+        "from typing_extensions import TypedDict, NotRequired # noqa: F401\n"
```

Comment:
Not a big deal, but in principle we can avoid `noqa: F401` if we look at `class_string` and drop any imports we don't need.